kotlin/android/app/src/main/java/dev/firezone/android/tunnel/TunnelManager.kt

```diff
@@ -151,7 +157,9 @@ internal class TunnelManager @Inject constructor(
                 sessionPtr = TunnelSession.connect(
                     controlPlaneUrl = BuildConfig.CONTROL_PLANE_URL,
                     token = config.token,
-                    externalId = Settings.Secure.ANDROID_ID,
+                    deviceId = deviceId(),
+                    logDir = appContext.filesDir.absolutePath,
```

Comment:
```suggestion
                    logDir = appContext.cacheDir,
```

Logs can be stored in cache. `cacheDir` returns an absolute path to app's internal cache dir.